README.md

```diff
@@ -371,22 +371,24 @@ Here's an example of building a small form with Concur -
 -- This is like Elm's State
 type Form =
   { name :: String
-  , rememberMe :: Bool
+  , rememberMe :: Boolean
   }
 
 -- This is like Elm's Action
 data FormAction
   = Name String
-  | RememberMe Bool
+  | RememberMe Boolean
   | Submit
 
 formWidget :: Form -> Widget HTML Form
 formWidget form = do
   -- This is like Elm's view function
   res <- div'
-    [ Name <$> input [_type "text", value form.name, unsafeTargetValue <$> onChange] []
-    , RememberMe <$> input [_type "checkbox", checked form.rememberMe, onChecked] []
-    , Submit <$ button [value "Submit", onClick] []
+    [ Name <$> input [ _type "text", value form.name,
```

Comment:
This change is entirely due to whitespace. Can you remove it?

Reply:
Oh I see, that you have removed the `[]` from the `input`s. However you also removed it from the `button` by accident. So you should put it back in. And please remove the whitespace changes as that obscures things.